libminifi/test/unit/BackTraceTests.cpp

```diff
@@ -103,9 +103,13 @@ TEST_CASE("BT2", "[TPT2]") {
 
   std::vector<BackTrace> traces = pool.getTraces();
   for (const auto &trace : traces) {
+    std::cerr << "Thread name: " << trace.getName() << std::endl;
     const auto &trace_strings = trace.getTraces();
 #ifdef HAS_EXECINFO
     REQUIRE(trace_strings.size() > 2);
+    for (const auto& trace_string : trace_strings) {
+      std::cerr << " - " << trace_string << std::endl;
```

Comment:
Is this intended?

Reply:
Yep, it doesn't take up much stderr output, but helps immensely in verifying the actual stack traces. With different types of symbolic information, the output can be very different, so I used this test to look at different cases on different platforms, and I think it can still be useful for that in the future.

Reply:
Okay, it doesn't here.